polkadot-parachain/src/command.rs

```diff
@@ -85,7 +85,7 @@ fn runtime(id: &str) -> Runtime {
 	} else if id.starts_with("westmint") {
 		Runtime::Westmint
 	} else if id.starts_with("penpal") {
-		Runtime::Penpal(para_id.unwrap())
+		Runtime::Penpal(para_id.unwrap_or(ParaId::new(0)))
```

Comment:
At the point where this is used downstream the para id is not used as it's already been parsed and stripped off earlier.